seventh-lesson/src/task27/CardRunner.java

```diff
@@ -0,0 +1,48 @@
+package task27;
+
+import task27.BelCard.BelCardCredit;
+import task27.BelCard.BelCardDebet;
+import task27.Visa.VisaElectron;
+import task27.Visa.VisaGold;
+
+public class CardRunner {
+    public static void main(String[] args) {
+
+//        BelCardCredit credit = new BelCardCredit();
```

Comment:
???
doesn't work?

Reply:
Sorry, I forgot to remove. Commented out so that unnecessary information is not displayed in the console at the moment